docs/sections/user_guide/yaml/components/mpas.rst

```diff
@@ -75,20 +75,11 @@ namelist:
 
 Supports ``base_file:`` and ``update_values:`` blocks (see the :ref:`updating_values` for details).
 
-.. include:: ../../../../shared/validate_namelist.rst
+.. include:: /shared/validate_namelist.rst
 
 run_dir:
 ^^^^^^^^
 
 The path to the run directory.
 
-streams:
-^^^^^^^^
-
-  **path:**
-
-  The path to the base ``streams.atmosphere`` file that comes from the MPAS build.
-
-  **values:**
-
-  The set of key-value pairs that will render the appropriate XML entries in the streams input file.
+.. include:: /shared/mpas_streams.rst
```

Comment:
The description of the `streams:` block is identical for MPAS Atm and MPAS Init, so has been factored out to a shared RST file.